pkgs/tools/package-management/nix-doc/default.nix

```diff
@@ -2,21 +2,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nix-doc";
-  version = "0.5.3";
+  version = "0.5.4";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "lf-";
     repo = "nix-doc";
-    sha256 = "sha256-H8FNOOjHMUW2wIUfoDhS3eH2AgxxD0LAuX4J9SJyJhg=";
+    sha256 = "sha256-bijcLIRBfoqirwz98Q3uQjHXSOaaqZECfav4TUvCuxg=";
   };
 
   doCheck = true;
   buildInputs = [ boost nix ];
 
   nativeBuildInputs = [ pkg-config ];
 
-  cargoSha256 = "sha256-BTMFoZ8HqbgUXkVyydOkcQZ10TLE8KsGRdt+xhBKJVc=";
+  cargoSha256 = "sha256-LpcAMsBeNa2GDGN7+9rTtkQluPfHSnAxanRtDtRahzc=";
 
   meta = with lib; {
     description = "An interactive Nix documentation tool";
```

Comment:
btw meta.platforms.unix is the default for buildRustPackage and should not be repeated.